boa_engine/src/vm/call_frame/abrupt_record.rs

```diff
@@ -0,0 +1,75 @@
+//! Implements an `AbruptCompletionRecord` struct for `CallFrame` tracking.
+
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub(crate) enum AbruptKind {
+    Continue,
+    Break,
+    Throw,
+}
+
+/// The `AbruptCompletionRecord` tracks the current `AbruptCompletion` and target address of completion.
+#[derive(Clone, Copy, Debug)]
+pub(crate) struct AbruptCompletionRecord {
+    kind: AbruptKind,
+    target: u32,
+}
+
+/// ---- `AbruptCompletionRecord` initialization methods ----
+impl AbruptCompletionRecord {
+    /// Creates an `AbruptCompletionRecord` for an abrupt `Break`.
+    pub(crate) const fn create_break_completion() -> Self {
```

Comment:
Would be better to shorten the names to `break`, `continue` and `throw`, seeing that the getters are preceded by `is_*`; it avoids repetition and makes remembering the name of the constructor a bit easier.